### ISSUE TITLE
Latest TikTok version kills functionality 

### ISSUE THREAD

**Issue by chrisjpn** [haoict/tiktok-god#60]
Latest version has removed all functionality of TT God.

**Comment by haoict** [comment]
Hi, could you provide more information? specially which feature has been broken?
(as I just checked, hide UI seems broken, others are still working fine)

**Comment by chrisjpn** [comment]
Hey, sure.  Basically all functionality has disappeared leaving only the basic TikTok features.  There’s no progress bar, can only download what TT allows with the watermark, can’t hide the UI and the download button has disappeared.  Happened as soon as I updated the app.

**Comment by haoict** [comment]
Pretty sure that it's not the problem with the tweak itself.
try to reinstall it.

**Comment by faryadoghez** [comment]
there is a problem in downloding,removed and installed again but still there is a problem,those video will download that tik tok allowed to download without watermark

**Comment by chrisjpn** [comment]
Yeah I tried removing and reinstalling but it made no difference.  I had to downgrade TikTok to restore functionality.

**Comment by faryadoghez** [comment]
yes from 4 days ago untill now i got same problem !

**Comment by megatech1966** [comment]
Save all videos no longer works. Remove ads no longer works.

**Comment by haoict** [comment]
> Save all videos no longer works. Remove ads no longer works.

I'm aware of this. In the mean time, you can turn on "Show Top Right Corner Download Button" in Settings App to download
